src/static/css/pages/sub-pages.less

```diff
@@ -3,10 +3,51 @@
    Sub-Pages pages at /sub-pages/*.
    ========================================================================== */
 
+.green-bullets {
+    li {
+        color: @black;
+        list-style: none;
+    }
+
+    li:before {
+        position: relative;
+        top: .1em;
+        margin-left: -1em;
+        padding-right: .3em;
+
+        color: @green;
+        //Square bullet point
+        content: "▪ ";
+        font-size: 1.5em;
+
+
+    }
+}
+
+
 .sub-page {
     .webfont-regular();
+
+    a[href $='.pdf'] {
```

Comment:
It would be more desirable to add `class="icon-link icon-link__download"` to the markup for these links. (And also wrap the interior text in `<span class="icon-link_text">`.)